bench/rttanalysis: TestBenchmarkExpectation failed [expected Jobs/cancel_job to perform KV lookups in [12, 12], got 14]

bench/rttanalysis.TestBenchmarkExpectation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18440703?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18440703?buildTab=artifacts#/) on master @ [c603477127055ef6ad5d2b3149b214e004efb5ed](https://github.com/cockroachdb/cockroach/commits/c603477127055ef6ad5d2b3149b214e004efb5ed):


```
    validate_benchmark_data.go:117: fail: expected Jobs/cancel_job to perform KV lookups in [12, 12], got 14
    validate_benchmark_data.go:120: success: expected ORMQueries/information_schema._pg_index_position to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:120: success: expected Jobs/non_admin_show_jobs to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:120: success: expected ORMQueries/hasura_column_descriptions_modified to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:120: success: expected Jobs/show_jobs to perform KV lookups in [3, 5], got 3
    validate_benchmark_data.go:120: success: expected Jobs/crdb_internal.system_jobs to perform KV lookups in [3, 3], got 3
    validate_benchmark_data.go:120: success: expected Jobs/jobs_page_type_filtered_no_matches to perform KV lookups in [1, 3], got 1
    validate_benchmark_data.go:120: success: expected Jobs/jobs_page_type_filtered to perform KV lookups in [3, 3], got 3
    validate_benchmark_data.go:120: success: expected Jobs/jobs_page_latest_50 to perform KV lookups in [3, 3], got 3
    validate_benchmark_data.go:120: success: expected Jobs/jobs_page_default to perform KV lookups in [3, 5], got 3
    validate_benchmark_data.go:120: success: expected ORMQueries/hasura_column_descriptions_8_tables to perform KV lookups in [20, 20], got 20
    validate_benchmark_data.go:120: success: expected ORMQueries/prisma_types_4 to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:120: success: expected ORMQueries/npgsql_types to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:120: success: expected ORMQueries/prisma_column_descriptions_updated to perform KV lookups in [3, 3], got 4
    validate_benchmark_data.go:120: success: expected ORMQueries/npgsql_fields to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:120: success: expected ORMQueries/asyncpg_types to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:120: success: expected Discard/DISCARD_ALL,_no_tables to perform KV lookups in [0, 0], got 0
    validate_benchmark_data.go:120: success: expected ORMQueries/prisma_types_16 to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:120: success: expected Discard/DISCARD_ALL,_1_tables_in_1_db to perform KV lookups in [16, 16], got 16
    validate_benchmark_data.go:120: success: expected DropTable/drop_3_tables to perform KV lookups in [19, 19], got 19
    validate_benchmark_data.go:120: success: expected DropView/drop_3_views to perform KV lookups in [17, 17], got 17
    validate_benchmark_data.go:120: success: expected DropView/drop_1_view to perform KV lookups in [16, 16], got 16
    validate_benchmark_data.go:120: success: expected DropTable/drop_2_tables to perform KV lookups in [17, 17], got 17
    validate_benchmark_data.go:120: success: expected DropTable/drop_1_table to perform KV lookups in [15, 15], got 15
    validate_benchmark_data.go:120: success: expected DropView/drop_2_views to perform KV lookups in [17, 17], got 17
    validate_benchmark_data.go:120: success: expected RevokeRole/revoke_1_role to perform KV lookups in [15, 15], got 15
    validate_benchmark_data.go:120: success: expected AlterRole/alter_role_with_3_options to perform KV lookups in [27, 27], got 27
    validate_benchmark_data.go:120: success: expected Grant/grant_all_on_2_tables to perform KV lookups in [10, 10], got 10
    validate_benchmark_data.go:120: success: expected Grant/grant_all_on_1_table to perform KV lookups in [10, 10], got 10
    validate_benchmark_data.go:120: success: expected GrantRole/grant_1_role to perform KV lookups in [17, 17], got 17
    validate_benchmark_data.go:120: success: expected Grant/grant_all_on_3_tables to perform KV lookups in [10, 10], got 10
    validate_benchmark_data.go:120: success: expected AlterRole/alter_role_with_1_option to perform KV lookups in [16, 16], got 16
    validate_benchmark_data.go:120: success: expected ORMQueries/liquibase_migrations to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:120: success: expected AlterRole/alter_role_with_2_options to perform KV lookups in [19, 19], got 19
    validate_benchmark_data.go:120: success: expected UseManyRoles/use_2_roles to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:120: success: expected ORMQueries/column_descriptions_json_agg to perform KV lookups in [6, 6], got 6
    validate_benchmark_data.go:120: success: expected ORMQueries/liquibase_migrations_on_multiple_dbs to perform KV lookups in [8, 8], got 8
    validate_benchmark_data.go:120: success: expected ORMQueries/prisma_column_descriptions to perform KV lookups in [133, 133], got 133
    validate_benchmark_data.go:120: success: expected UseManyRoles/use_50_roles to perform KV lookups in [3, 3], got 3
    validate_benchmark_data.go:73: see the -rewrite flag to re-run the benchmarks and adjust the expectations
    validate_benchmark_data.go:74: usage: ./dev test --rewrite pkg/bench/rttanalysis -f=TestBenchmarkExpectation/ --test-args '-rewrite-iterations=N'
    validate_benchmark_data.go:107: -- test log scope end --
--- FAIL: TestBenchmarkExpectation (131.16s)
--- FAIL: TestBenchmarkExpectation (131.76s)
=== RUN   TestBenchmarkExpectation/Jobs
    --- FAIL: TestBenchmarkExpectation/Jobs (94.32s)
=== RUN   TestBenchmarkExpectation/Jobs/cancel_job
    rtt_analysis_bench.go:224: Jobs/cancel_job: got 14, expected 12
    rtt_analysis_bench.go:230: wrote jaeger trace to /artifacts/tmp/_tmp/b03b0378e25aebc8fcd368039859e7a0/logTestBenchmarkExpectation3181587019/Jobs_cancel_job.jaeger.json
        --- FAIL: TestBenchmarkExpectation/Jobs/cancel_job (91.41s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #126750 bench/rttanalysis: TestBenchmarkExpectation failed [Jobs/jobs_page_type_filtered_no_matches] [A-jobs C-test-failure O-robot P-2 T-jobs branch-release-24.1]
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBenchmarkExpectation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-46342

bench/rttanalysis.TestBenchmarkExpectation [failed](https://tanzanite.cluster.engflow.com/invocations/default/c2e434f4-06dc-4674-8103-33fda8c7137b?testReportRun=18&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvYmVuY2gvcnR0YW5hbHlzaXM6cnR0YW5hbHlzaXNfdGVzdA==) on master @ [358c9a5ada822ab2cdc0aa09baa2f08028fadc3c](https://github.com/cockroachdb/cockroach/commits/358c9a5ada822ab2cdc0aa09baa2f08028fadc3c):


```
    validate_benchmark_data.go:120: success: expected AlterRole/alter_role_with_2_options to perform KV lookups in [19, 19], got 19
    validate_benchmark_data.go:120: success: expected RevokeRole/revoke_1_role to perform KV lookups in [15, 15], got 15
    validate_benchmark_data.go:120: success: expected ORMQueries/django_column_introspection_1_table to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:120: success: expected ORMQueries/django_column_introspection_4_tables to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:120: success: expected ORMQueries/django_column_introspection_8_tables to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:120: success: expected ORMQueries/django_table_introspection_1_table to perform KV lookups in [6, 6], got 5
    validate_benchmark_data.go:120: success: expected ORMQueries/django_table_introspection_8_tables to perform KV lookups in [6, 6], got 5
    validate_benchmark_data.go:120: success: expected ORMQueries/django_comment_introspection_with_comments to perform KV lookups in [6, 6], got 5
    validate_benchmark_data.go:120: success: expected ORMQueries/activerecord_type_introspection_query to perform KV lookups in [3, 3], got 3
    validate_benchmark_data.go:120: success: expected ORMQueries/pg_type to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:120: success: expected ORMQueries/pg_class to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:120: success: expected ORMQueries/pg_namespace to perform KV lookups in [3, 3], got 3
    validate_benchmark_data.go:120: success: expected ORMQueries/pg_attribute to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:120: success: expected ORMQueries/introspection_description_join to perform KV lookups in [4, 4], got 5
    validate_benchmark_data.go:120: success: expected ORMQueries/has_schema_privilege to perform KV lookups in [0, 0], got 0
    validate_benchmark_data.go:120: success: expected ORMQueries/has_sequence_privilege to perform KV lookups in [0, 0], got 0
    validate_benchmark_data.go:120: success: expected ORMQueries/has_table_privilege to perform KV lookups in [0, 0], got 0
    validate_benchmark_data.go:120: success: expected ORMQueries/has_column_privilege_using_attnum to perform KV lookups in [0, 0], got 0
    validate_benchmark_data.go:120: success: expected ORMQueries/has_column_privilege_using_column_name to perform KV lookups in [0, 0], got 0
    validate_benchmark_data.go:120: success: expected ORMQueries/pg_my_temp_schema to perform KV lookups in [3, 3], got 3
    validate_benchmark_data.go:120: success: expected ORMQueries/pg_my_temp_schema_multiple_times to perform KV lookups in [3, 3], got 3
    validate_benchmark_data.go:120: success: expected ORMQueries/pg_is_other_temp_schema to perform KV lookups in [6, 6], got 6
    validate_benchmark_data.go:120: success: expected ORMQueries/pg_is_other_temp_schema_multiple_times to perform KV lookups in [6, 6], got 6
    validate_benchmark_data.go:120: success: expected ORMQueries/information_schema._pg_index_position to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:120: success: expected ORMQueries/hasura_column_descriptions to perform KV lookups in [5, 5], got 6
    validate_benchmark_data.go:120: success: expected ORMQueries/hasura_column_descriptions_8_tables to perform KV lookups in [20, 20], got 20
    validate_benchmark_data.go:120: success: expected ORMQueries/hasura_column_descriptions_modified to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:120: success: expected ORMQueries/column_descriptions_json_agg to perform KV lookups in [6, 6], got 6
    validate_benchmark_data.go:120: success: expected ORMQueries/prisma_column_descriptions_updated to perform KV lookups in [3, 3], got 4
    validate_benchmark_data.go:120: success: expected ORMQueries/prisma_types_4 to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:120: success: expected ORMQueries/prisma_column_descriptions to perform KV lookups in [133, 133], got 133
    validate_benchmark_data.go:120: success: expected ORMQueries/prisma_types_16 to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:120: success: expected ORMQueries/npgsql_types to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:120: success: expected ORMQueries/npgsql_fields to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:120: success: expected ORMQueries/asyncpg_types to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:120: success: expected AlterTableAddCheckConstraint/alter_table_add_1_check_constraint to perform KV lookups in [15, 15], got 15
    validate_benchmark_data.go:120: success: expected ORMQueries/liquibase_migrations to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:120: success: expected AlterTableAddCheckConstraint/alter_table_add_2_check_constraints to perform KV lookups in [15, 15], got 15
    validate_benchmark_data.go:73: see the -rewrite flag to re-run the benchmarks and adjust the expectations
    validate_benchmark_data.go:74: usage: ./dev test --rewrite pkg/bench/rttanalysis -f=TestBenchmarkExpectation/ --test-args '-rewrite-iterations=N'
    validate_benchmark_data.go:107: -- test log scope end --
test logs left over in: outputs.zip/logTestBenchmarkExpectation2993277443
--- FAIL: TestBenchmarkExpectation (420.57s)
--- FAIL: TestBenchmarkExpectation (420.81s)
=== RUN   TestBenchmarkExpectation/ORMQueries
    --- FAIL: TestBenchmarkExpectation/ORMQueries (419.02s)
=== RUN   TestBenchmarkExpectation/ORMQueries/liquibase_migrations_on_multiple_dbs
    rtt_analysis_bench.go:224: ORMQueries/liquibase_migrations_on_multiple_dbs: got 6.5, expected 8
    rtt_analysis_bench.go:230: wrote jaeger trace to /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/testlogs/pkg/bench/rttanalysis/rttanalysis_test/run_18_of_25/test.outputs/logTestBenchmarkExpectation2993277443/ORMQueries_liquibase_migrations_on_multiple_dbs.jaeger.json
        --- FAIL: TestBenchmarkExpectation/ORMQueries/liquibase_migrations_on_multiple_dbs (124.52s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=18</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #126750 bench/rttanalysis: TestBenchmarkExpectation failed [Jobs/jobs_page_type_filtered_no_matches] [A-jobs C-test-failure O-robot P-2 T-jobs branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBenchmarkExpectation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

bench/rttanalysis.TestBenchmarkExpectation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18541880?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18541880?buildTab=artifacts#/) on master @ [f7c32ec5a3f902bfcebb29cf8e9794092d9c2a5b](https://github.com/cockroachdb/cockroach/commits/f7c32ec5a3f902bfcebb29cf8e9794092d9c2a5b):


```
    validate_benchmark_data.go:120: success: expected ORMQueries/hasura_column_descriptions_8_tables to perform KV lookups in [20, 20], got 20
    validate_benchmark_data.go:120: success: expected DropRole/drop_2_roles to perform KV lookups in [43, 43], got 43
    validate_benchmark_data.go:120: success: expected DropView/drop_2_views to perform KV lookups in [17, 17], got 17
    validate_benchmark_data.go:120: success: expected SystemDatabaseQueries/select_system.users_with_empty_database_Name to perform KV lookups in [1, 1], got 1
    validate_benchmark_data.go:120: success: expected RevokeRole/revoke_1_role to perform KV lookups in [15, 15], got 15
    validate_benchmark_data.go:120: success: expected RevokeRole/revoke_2_roles to perform KV lookups in [17, 17], got 17
    validate_benchmark_data.go:120: success: expected ORMQueries/pg_namespace to perform KV lookups in [3, 3], got 3
    validate_benchmark_data.go:120: success: expected Revoke/revoke_all_on_2_tables to perform KV lookups in [10, 10], got 10
    validate_benchmark_data.go:120: success: expected Grant/grant_all_on_2_tables to perform KV lookups in [10, 10], got 10
    validate_benchmark_data.go:120: success: expected ORMQueries/column_descriptions_json_agg to perform KV lookups in [6, 6], got 6
    validate_benchmark_data.go:120: success: expected VirtualTableQueries/select_crdb_internal.tables_with_1_fk to perform KV lookups in [1, 1], got 1
    validate_benchmark_data.go:120: success: expected ORMQueries/django_table_introspection_1_table to perform KV lookups in [6, 6], got 5
    validate_benchmark_data.go:120: success: expected ORMQueries/asyncpg_types to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:120: success: expected ORMQueries/django_table_introspection_8_tables to perform KV lookups in [6, 6], got 5
    validate_benchmark_data.go:120: success: expected VirtualTableQueries/select_crdb_internal.invalid_objects_with_1_fk to perform KV lookups in [1, 1], got 1
    validate_benchmark_data.go:120: success: expected ORMQueries/information_schema._pg_index_position to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:120: success: expected Truncate/truncate_2_column_2_rows to perform KV lookups in [20, 20], got 19
    validate_benchmark_data.go:120: success: expected GenerateObjects/generate_10x10_schemas_and_tables_in_existing_db to perform KV lookups in [16, 16], got 16
    validate_benchmark_data.go:120: success: expected Truncate/truncate_1_column_1_row to perform KV lookups in [20, 20], got 19
    validate_benchmark_data.go:120: success: expected Truncate/truncate_2_column_0_rows to perform KV lookups in [20, 20], got 19
    validate_benchmark_data.go:120: success: expected VirtualTableQueries/virtual_table_cache_with_point_lookups to perform KV lookups in [5, 5], got 5
    validate_benchmark_data.go:120: success: expected Truncate/truncate_2_column_1_rows to perform KV lookups in [20, 20], got 19
    validate_benchmark_data.go:120: success: expected Truncate/truncate_1_column_2_rows to perform KV lookups in [20, 20], got 19
    validate_benchmark_data.go:120: success: expected DropSequence/drop_2_sequences to perform KV lookups in [17, 17], got 17
    validate_benchmark_data.go:120: success: expected Truncate/truncate_1_column_0_rows to perform KV lookups in [20, 20], got 19
    validate_benchmark_data.go:120: success: expected Jobs/cancel_job to perform KV lookups in [12, 12], got 13
    validate_benchmark_data.go:120: success: expected CreateRole/create_role_with_no_options to perform KV lookups in [27, 27], got 27
    validate_benchmark_data.go:120: success: expected Jobs/non_admin_crdb_internal.system_jobs to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:120: success: expected CreateRole/create_role_with_1_option to perform KV lookups in [26, 26], got 26
    validate_benchmark_data.go:120: success: expected CreateRole/create_role_with_3_options to perform KV lookups in [32, 32], got 32
    validate_benchmark_data.go:120: success: expected CreateRole/create_role_with_2_options to perform KV lookups in [29, 29], got 29
    validate_benchmark_data.go:120: success: expected ShowGrants/grant_4_roles to perform KV lookups in [6, 6], got 6
    validate_benchmark_data.go:120: success: expected ORMQueries/liquibase_migrations to perform KV lookups in [4, 4], got 4
    validate_benchmark_data.go:120: success: expected UseManyRoles/use_2_roles to perform KV lookups in [2, 2], got 2
    validate_benchmark_data.go:120: success: expected UseManyRoles/use_50_roles to perform KV lookups in [0, 0], got 0
    validate_benchmark_data.go:120: success: expected Jobs/jobs_page_default to perform KV lookups in [3, 5], got 3
    validate_benchmark_data.go:120: success: expected ORMQueries/prisma_column_descriptions to perform KV lookups in [133, 133], got 133
    validate_benchmark_data.go:120: success: expected ORMQueries/liquibase_migrations_on_multiple_dbs to perform KV lookups in [8, 8], got 8
    validate_benchmark_data.go:73: see the -rewrite flag to re-run the benchmarks and adjust the expectations
    validate_benchmark_data.go:74: usage: ./dev test --rewrite pkg/bench/rttanalysis -f=TestBenchmarkExpectation/ --test-args '-rewrite-iterations=N'
    validate_benchmark_data.go:107: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/b03b0378e25aebc8fcd368039859e7a0/logTestBenchmarkExpectation3010421285
--- FAIL: TestBenchmarkExpectation (140.99s)
=== RUN   TestBenchmarkExpectation/Jobs/resume_job
    test_server_shim.go:141: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
    rtt_analysis_bench.go:224: Jobs/resume_job: got 14, expected 12
    rtt_analysis_bench.go:230: wrote jaeger trace to /artifacts/tmp/_tmp/b03b0378e25aebc8fcd368039859e7a0/logTestBenchmarkExpectation3010421285/Jobs_resume_job.jaeger.json
        --- FAIL: TestBenchmarkExpectation/Jobs/resume_job (47.92s)
=== RUN   TestBenchmarkExpectation/Jobs
    --- FAIL: TestBenchmarkExpectation/Jobs (116.62s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #126750 bench/rttanalysis: TestBenchmarkExpectation failed [Jobs/jobs_page_type_filtered_no_matches] [A-jobs C-test-failure O-robot P-2 T-jobs branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBenchmarkExpectation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/bench/rttanalysis/testdata/benchmark_expectations
+++ b/pkg/bench/rttanalysis/testdata/benchmark_expectations
@@ -61,7 +61,7 @@ exp,benchmark
 10,Grant/grant_all_on_3_tables
 17,GrantRole/grant_1_role
 21,GrantRole/grant_2_roles
-12,Jobs/cancel_job
+12-15,Jobs/cancel_job
 3,Jobs/crdb_internal.system_jobs
 3-5,Jobs/jobs_page_default
 3,Jobs/jobs_page_latest_50
@@ -69,8 +69,8 @@ exp,benchmark
 1-3,Jobs/jobs_page_type_filtered_no_matches
 4,Jobs/non_admin_crdb_internal.system_jobs
 4,Jobs/non_admin_show_jobs
-14,Jobs/pause_job
-12,Jobs/resume_job
+12-15,Jobs/pause_job
+12-15,Jobs/resume_job
 3,Jobs/show_job
 3-5,Jobs/show_jobs
 3,ORMQueries/activerecord_type_introspection_query

